pom.xml

```diff
@@ -66,15 +67,16 @@
         <dependency>
             <groupId>com.palmergames.bukkit.towny</groupId>
             <artifactId>Towny</artifactId>
-            <version>0.96.0.0</version>
-            <scope>provided</scope>
+            <version>0.96.1.0</version>
+            <scope>system</scope>
+            <systemPath>${project.basedir}/deps/Towny-0.96.1.0.jar</systemPath>
```

Comment:
set scope back to provided and remove <systemPath>

Reply:
Done